c2corg_ui/templates/image/edit.html

```diff
@@ -193,17 +195,20 @@ <h2 class="heading show-phone" translate>camera settings</h2>
             </div>
 
             ## EXPOSURE TIME
-            <div class="form-group half" ng-class="{ 'has-error': editForm.exposure_time.$invalid, 'has-success': editForm.exposure_time.$valid && image.exposure_time }">
+            <div class="form-group half" ng-class="{ 'has-error': editForm.exposure_time.$invalid, 'has-success': editForm.exposure_time.$valid && image.exposure_time}">
               <label translate>exposure_time</label>
-              <div class="input-group">
-                <span class="input-group-addon">1/</span>
-                <input type="number" min="1" name="exposure_time" ng-model="image.exposure_time" class="form-control">
-                <span class="input-group-addon">s</span>
+              <div class="input-group flex">
+                {{editCtrl.convertExposureTime(image.exposure_time)}}
+                <input type="number" min="0" max="30" name="exposure_time" ng-model="image.exposure_time" ng-change="editCtrl.convertExposureTime(image.exposure_time)" class="form-control"
+                       uib-tooltip="{{'You can type 0.003125 and it will be converted to 1/320s' | translate}}">
```

Comment:
i would have no idea that you should type the value in this format if there were no help message :P